lighthouse-core/gather/driver.js

```diff
@@ -833,6 +833,16 @@ class Driver {
     const promiseArr = urlPatterns.map(url => this.sendCommand('Network.addBlockedURL', {url}));
     return Promise.all(promiseArr);
   }
+
+  handleJavaScriptDialogs() {
```

Comment:
My first reaction was that maybe we should use `this.once` instead of `this.on`. IOW, dismiss the first prompt we see and ignore the rest. Thinking towards the future when we have auditing after the page load (SPAs), this will dismiss all prompts the page throws up. That might not be something we want to do.

That said, it's a ways out and this `on` is probably safer for now.